content/2022-munich/speakers/filip-petkovski.md

```diff
@@ -5,6 +5,6 @@ title: "Filip Petkovski"
 
 Working at Shopify
 
-Filip is a production engineer at Shopify, currently working on monitoring the monitoring stack for Shopify’s infrastructure. He is a kube-state-metrics maintainer and a member of the Thanos team.
+Filip is a production engineer at Shopify, currently working the monitoring stack for Shopify’s infrastructure. He is a kube-state-metrics maintainer and a member of the Thanos team.
```

Comment:
Didn't this remove one word too much? "working the" should still be "working on the", right?

Reply:
Ah this is right. Hopefully this is the last fix: https://github.com/promcon/website/pull/236